core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-row/template-builder-row.component.html

```diff
@@ -23,4 +23,7 @@
             (deleteConfirmed)="deleteRow()"
         ></dotcms-remove-confirm-dialog>
     </div>
+    <dotcms-template-builder-background-columns
+        [show]="isResizing"
```

Comment:
```suggestion
        [ngStyle]={{opacity: isResizing ? 1 : 0}}
```

This code might not work, but is the idea.